codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/CodegenVisitor.java

```diff
@@ -220,6 +220,8 @@ void execute() {
 
         LOGGER.fine("Generating go.mod file");
         GoModGenerator.writeGoMod(settings, fileManifest, SymbolDependency.gatherDependencies(dependencies.stream()));
+        LOGGER.fine("Generating generated.json file");
+        ManifestWriter.writeManifest(settings, fileManifest, dependencies);
```

Comment:
next step is to remove the `go.mod` generation, once we have tooling to use this generated.json file correct?

Reply:
Yes that is correct, I want to have the pieces together in the SDK repo for generating the `go.mod` from the `generated.json` first before we remove it from the generated artifacts here. That way we don't break out existing tooling. So there will be a follow-up PR to remove the `go.mod` when we are ready to do so.